src/components/Modals/ProposeBudgetModal.js

```diff
@@ -0,0 +1,264 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import PropTypes from 'prop-types';
+
+// COMPONENTS
+import { useDispatch, useSelector } from 'react-redux';
+import { BN } from '@polkadot/util';
+import ModalRoot from './ModalRoot';
+import Button from '../Button/Button';
+import styles from './styles.module.scss';
+import { SelectInput, TextInput } from '../InputComponents';
+import { congressActions } from '../../redux/actions';
+import { congressSelectors } from '../../redux/selectors';
+import { isValidSubstrateAddress, parseAssets } from '../../utils/walletHelpers';
+import {
+  closestNumberToZeroNotInArray,
+  extractItemsFromObject, getOperationType, OperationsType,
+} from '../../utils/councilHelper';
+import useCongressExecutionBlock from '../../hooks/useCongressExecutionBlock';
+import InputSearch from '../InputComponents/InputSearchAddressName';
+import RemarkForm from '../WalletCongresSenate/RemarkForm';
+
+const defaultValueSpending = {
+  currency: null,
+  amount: null,
+  remark: null,
+  optionsInput: [{
+    value: 'LLD',
+    display: 'Liberland Dolar (LLD)',
+    index: 0,
+  }],
+  indexItem: 0,
+};
+
+const collectDataItemList = [
+  'project',
+  'description',
+  'recipient',
+  'transfer',
+  'select',
+  'category',
+  'supplier',
+];
+
+function ProposeBudgetModal({
+  closeModal,
+}) {
+  const dispatch = useDispatch();
+  const allBalance = useSelector(congressSelectors.allBalance);
+  const [itemsInList, setItemsInList] = useState([0]);
+  const [spendings, setSpendings] = useState([defaultValueSpending]);
+  const [indexAssets, setIndexAssets] = useState({ 0: OperationsType.LLD });
+
+  const {
+    handleSubmit,
+    register,
+    trigger,
+    setValue,
+    formState: { errors, isValid },
+    unregister,
+  } = useForm({
+    mode: 'onChange',
+    defaultValues: {
+      votingDays: '7',
+    },
+  });
+
+  const executionBlock = useCongressExecutionBlock(7);
+
+  const onSubmit = (data) => {
+    if (!isValid) return;
+    const bugetProposalItems = extractItemsFromObject(data, spendings[0].optionsInput);
+    dispatch(congressActions.congressBudgetPropose.call({ bugetProposalItems, executionBlock }));
+  };
+
+  const optionsInputDefault = useMemo(() => allBalance.map((item) => {
+    const { metadata, index, balance } = item;
+    const { symbol, name, decimals } = metadata;
+    const operationType = getOperationType(symbol);
```

Comment:
We can't abuse symbol for choosing transfer type here.
1. Asset symbols are chosen by people, anyone can create asset with a symbol "LLD" which would break this system here
2. It's just error-prone and I'd guess this may be related to error reported by @DorianSternVukotic (where LLD transfer used asset.transfer onchain)